cinnabar/tests/test_stats.py

```diff
@@ -145,3 +145,24 @@ def test_correlation_positive(fe_map):
         assert (
             0.5 < bss["mle"] < 0.9
         ), f"Correlation must be positive for this data. {stat} is {bss['mle']}"
+
+def test_confidence_intervals(fe_map):
+    """
+    Test that the confidence intervals for RMSE and MUE contain the corresponding statistics.
+    Uses the example data in `cinnabar/data/example.csv`
+    """
+
+    nodes = fe_map.graph.nodes
+
+    x_data = np.asarray([n[1]["exp_DG"] for n in nodes(data=True)])
+    y_data = np.asarray([n[1]["calc_DG"] for n in nodes(data=True)])
+    xerr = np.asarray([n[1]["exp_dDG"] for n in nodes(data=True)])
+    yerr = np.asarray([n[1]["calc_dDG"] for n in nodes(data=True)])
+
+    # RMSE
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE")
+    assert bss['low'] < bss['mean'] < bss['high'], "The RMSE must lie withint the bootstrapped 95% CI"
```

Comment:
This is a good first test but it doesn't actually trigger either of the newly introduced bools for adding noise either in the x or y direction. Ideally we need a test for that, probably maybe a regression test (assuming enough bootstraps to ensure convergence)?

Reply:
I just changed the test to use the newly introduced bools and check whether the RMSE lies within the 95% CIs. 

I thought about adding a regression test -- was thinking I could add my data from #73 to `cinnabar/data/` and then create a test that checks whether that specific dataset passes the test, but I'm not using a standard cinnabar csv input file for that dataset, because its not the standard "predicted DDG" vs "experiment DDG", its "forward DDG" vs "reverse DDG." It would take a bit of time to properly format it and I'm not sure its essential to include, so I decided not to.

Reply:
Let me know if you think of better tests / feel free to add them yourself if you have time!